tests/src/test/scala/whisk/core/limits/ActionLimitsTests.scala

```diff
@@ -269,4 +269,29 @@ class ActionLimitsTests extends TestHelpers with WskTestHelpers {
             }
     }
 
+    it should "limit the length of HTTP req/resp bodies for debug messages to 1000 bytes" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val auth: Seq[String] = Seq("--auth", wskprops.authKey)
+            val name = "largeFileAction"
+            val debugMsg = "Body excedes 1000 bytes and will be truncated"
```

Comment:
Right, you wouldn't be able to check if a request was encoded correctly. Maybe adding a flag to allow for truncation would be correct path to take?